spharpy/samplings/samplings.py

```diff
@@ -564,15 +564,14 @@ def gaussian(n_points=None, n_max=None, radius=1.):
         n_max = int(n_max)
 
     # construct the sampling grid
-    legendre, weights = np.polynomial.legendre.leggauss(int(n_theta))
+    legendre, weights = np.polynomial.legendre.leggauss(n_max+1)
```

Comment:
```suggestion
    legendre, weights = np.polynomial.legendre.leggauss(int(n_theta))
```
i think this should be n_theta, otherwise n_theta could be wrong if points are set separatly.